### ISSUE TITLE
Related Images

### ISSUE THREAD

**Issue by Irepan8** [fanfare/googleimagesrestored#44]
Related Images isn't working on Google Chrome,

Would want to see if it can be fixed if possible,

**Comment by fanfare** [comment]
hi, more information is needed please. what's a search term you've tried that results in no related images? still seems to be working on my end

**Comment by Irepan8** [comment]
Mainly everything.

From memes to pizza. The related images don't seem to be working. Tried reinstalling the extension and didn't work.
Only extensions I've got apart from it is ublock type stuff and that's not why.
Maybe I've got DuckDuckGo as my search engine but that shouldn't really effect.
![pieee](https://user-images.githubusercontent.com/62076665/76663400-b11a6780-6578-11ea-8a28-149b1ee12e00.jpg)
Don't know if this screenshot helps.
